DepotDownloader/DepotDownloader.csproj

```diff
@@ -12,8 +12,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="protobuf-net" Version="3.2.16" />
+    <PackageReference Include="protobuf-net" Version="3.2.26" />
     <PackageReference Include="QRCoder" Version="1.4.3" />
-    <PackageReference Include="SteamKit2" Version="2.5.0-Beta.1" />
+    <!-- !Don't forget to change it to 3.0.0 once it's live -->
```

Comment:
Dependabot will hopefully do this for us.

Reply:
oh didn't realize it would still build with SteamKit 2.5.0, i'll switch it back to 2.5